packages/components/rollup.config.mjs

```diff
@@ -0,0 +1,78 @@
+import { Addon } from '@embroider/addon-dev/rollup';
+import { babel } from '@rollup/plugin-babel';
+import copy from 'rollup-plugin-copy';
+import scss from 'rollup-plugin-scss';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist',
+});
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints(['**/*.ts', '**/*.js', 'styles/@hashicorp/*.scss']),
+
+    // These are the modules that should get reexported into the traditional
+    // "app" tree. Things in here should also be in publicEntrypoints above, but
+    // not everything in publicEntrypoints necessarily needs to go here.
+    addon.appReexports(['**/*.ts', '**/*.js']),
```

Comment:
I think you may want to be more specific here - this is essentially replacing having an `app` folder. Before it was only the contents of the  `components/`, `helpers/` and `modifiers/` folders, but this is also now re-exporting things like `utils` which is causing a warning when I'm trying it locally:

```
WARNING in ../../../../node_modules/@hashicorp/design-system-components/dist/_app_/utils/hds-get-element-id.js 1:0-87
export 'default' (reexported as 'default') was not found in '@hashicorp/design-system-components/utils/hds-get-element-id' (possible exports: getElementId)
```

Reply:
updated in 875ac4a to only reexport the main 3 folders